io_scene_nif/nif_import.py

```diff
@@ -184,7 +183,8 @@ def import_root(self, root_block):
         #     self.animationhelper.import_text_keys(root_block)
 
         # read the NIF tree
-        self.active_obj_name = ""
+        Object.ACTIVE_OBJ_NAME = ""
```

Comment:
Can't we use bpy.context.active_object if we have to maintain a reference?

Reply:
This is gone in the subsequent PR